massgov/mayflower-patternlab
# use explicit height in iframe if provided

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

Use an explicit height for iframe embed if one is provided,. If not, default to responsive iframe behavior. 

## Related Issue / Ticket

- [JIRA issue](https://jira.state.ma.us/browse/DP-8441)
- [Github issue]()

## Steps to Test


1. Confirm `@atoms/09-media/iframe` has a set height
2. Confirm `@atoms/09-media/iframe-no-height-provided` maintains a 16:9 aspect ratio as the screen width changes.
3. Check that `@pages/pages-information-details` can have both types of iframes (aspect ratio and set height)

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
